docs/guides/guides/command-line.mdx

```diff
@@ -95,33 +95,31 @@ cypress run [options]
-| `--config`, `-c`               | [Specify configuration](#cypress-run-config-lt-config-gt)                                                                                                                                                               |
-| `--config-file`, `-C`          | [Specify configuration file](#cypress-run-config-file-lt-configuration-file-gt)                                                                                                                                         |
-| `--e2e`                        | Run end to end tests (default)                                                                                                                                                                                          |
-| `--env`, `-e`                  | [Specify environment variables](#cypress-run-env-lt-env-gt)                                                                                                                                                             |
-| `--group`                      | [Group recorded tests together under a single run](#cypress-run-group-lt-name-gt)                                                                                                                                       |
-| `--headed`                     | [Displays the browser instead of running headlessly](#cypress-run-headed)                                                                                                                                               |
-| `--headless`                   | [Hide the browser instead of running headed (default during `cypress run`)](#cypress-run-headless)                                                                                                                      |
-| `--help`, `-h`                 | Output usage information                                                                                                                                                                                                |
-| `--key`, `-k`                  | [Specify your secret record key](#cypress-run-record-key-lt-record-key-gt)                                                                                                                                              |
-| `--no-exit`                    | [Keep Cypress open after tests in a spec file run](#cypress-run-no-exit)                                                                                                                                                |
-| `--no-runner-ui`,              | [Hides the Cypress Runner UI](#cypress-run-no-runner-ui)                                                                                                                                                                |
```

Comment:
did the changes for `--no-runner-ui` get merged into the `v13` branch? I wonder if we have a bad merge here?

Reply:
@AtofStryker they did, https://github.com/cypress-io/cypress-documentation/pull/5315/commits/fc9304f692f30ba1597f77fda759ce3d06f00408

I had them here originally, but then dropped the commit in favor of the content already in the v13 branch